null.go

```diff
@@ -0,0 +1,41 @@
+package logr
+
+// Null returns a valid Logger that discards all messages logged to it.
+// It can be used whenever the caller is not interested in the logs.
+func Null() Logger {
+	return nullLogger{}
+}
+
+// IsNull checks if a Logger is a null logger as returned by Null.
+func IsNull(l Logger) bool {
```

Comment:
If we add `FromContextOrNull()` distinct from `FromContext()` then I think we don't need this - if you called OrNull you have waived the right to ask this question.  If you didn't call OrNull then you don't need to ask this question.

Reply:
Removing it